### PR TITLE
Follow-up to Docker Registry backed by GlusterFS (PR#1898)

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/1898

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@bfallonf @tnguyen-rh PTAL. Thanks!

File: install_config/storage_examples/gluster_backed_registry.adoc [hunk 1]
```diff
@@ -62,49 +68,58 @@ NAME            LABELS    STATUS    VOLUME       CAPACITY   ACCESSMODES   AGE
...
 ----
 
-
-
-NOTE: If either the *pvc* or *pv* failed to create or the pvc failed to bind, refer back to the
-link:../persistent_storage/persistent_storage_glusterfs.html[GlusterFS Persistent Storage] guide.
-*Do not* proceed until they initialize and the pvc status is `Bound`
+[NOTE]
+====
+If either the PVC or PV failed to create or the pvc failed to bind, refer back
```
Comment (bfallonf) [inline install_config/storage_examples/gluster_backed_registry.adoc]: s/pvc/PVC

File: install_config/storage_examples/gluster_backed_registry.adoc [hunk 2]
```diff
@@ -62,49 +68,58 @@ NAME            LABELS    STATUS    VOLUME       CAPACITY   ACCESSMODES   AGE
...
 
-In non-production cases where the dnsmasq server is located on the same node as the OpenShift master service,
-it is possible that the pods cannot resolve the host machines when mounting the volume,
-causing the `docker-registry-1-deploy` pod to error.
-This can happen when `dnsmasq.service` fails to start because of a collision with OpenShift DNS on port 53.
-To run the DNS server on the master host, some configurations needs to be changed.
+In non-production cases where the *dnsmasq* server is located on the same node
+as the {product-title} master service, it is possible that the pods cannot
+resolve the host machines when mounting the volume, causing the
+*docker-registry-1-deploy* pod to error.This can happen when *dnsmasq.service*
```
Comment (bfallonf) [inline install_config/storage_examples/gluster_backed_registry.adoc]: Is error a verb? Would it be better to say "causing errors in the *docker-registry-1-deploy* pod."?

File: install_config/storage_examples/gluster_backed_registry.adoc [hunk 3]
```diff
@@ -119,33 +134,38 @@ server=/17.30.172.in-addr.arpa/127.0.0.1#8053
...
 ====
 
-When pods are created, they receive a copy of */etc/resolve.conf*, which typically contains
-only the master DNS server so they can resolve external DNS requests.  To enable internal
-DNS resolution, insert the dnsmasq server to the beginning.  This way, dnsmasq will attempt to resolve
-requests internally first.
+When pods are created, they receive a copy of *_/etc/resolve.conf_*, which
+typically contains only the master DNS server so they can resolve external DNS
+requests. To enable internal DNS resolution, insert the *dnsmasq* server to the
+beginning. This way, *dnsmasq* will attempt to resolve requests internally
```
Comment (bfallonf) [inline install_config/storage_examples/gluster_backed_registry.adoc]: "To enable internal DNS resolution, insert the dnsmasq server to the beginning."

I'm guessing this is like any other resolv.conf file thing. Maybe:

"To enable internal DNS resolution, insert the dnsmasq server information at the top of the server list." or something.

File: install_config/storage_examples/gluster_backed_registry.adoc [hunk 4]
```diff
@@ -119,33 +134,38 @@ server=/17.30.172.in-addr.arpa/127.0.0.1#8053
...
 dnsConfig:
  bindAddress: 127.0.0.1:8053
 ----
 ====
 
-When pods are created, they receive a copy of */etc/resolve.conf*, which typically contains
-only the master DNS server so they can resolve external DNS requests.  To enable internal
-DNS resolution, insert the dnsmasq server to the beginning.  This way, dnsmasq will attempt to resolve
-requests internally first.
+When pods are created, they receive a copy of *_/etc/resolve.conf_*, which
```
Comment (bfallonf) [inline install_config/storage_examples/gluster_backed_registry.adoc]: Is it ever resolve.conf with an e? Everytime I see resolv.conf it's without an e. I know this was here already, but...I'm not sure.

File: install_config/storage_examples/gluster_backed_registry.adoc [hunk 5]
```diff
@@ -62,49 +68,58 @@ NAME            LABELS    STATUS    VOLUME       CAPACITY   ACCESSMODES   AGE
...
 === Pod Cannot Resolve the Volume Host
 
-In non-production cases where the dnsmasq server is located on the same node as the OpenShift master service,
-it is possible that the pods cannot resolve the host machines when mounting the volume,
-causing the `docker-registry-1-deploy` pod to error.
-This can happen when `dnsmasq.service` fails to start because of a collision with OpenShift DNS on port 53.
-To run the DNS server on the master host, some configurations needs to be changed.
+In non-production cases where the *dnsmasq* server is located on the same node
+as the {product-title} master service, it is possible that the pods cannot
+resolve the host machines when mounting the volume, causing the
```
Comment (bfallonf) [inline install_config/storage_examples/gluster_backed_registry.adoc]: the "it is" is expletive.

http://writing.wisc.edu/Handbook/CCS_expletive.html

Maybe:

"pods might not resolve to the host machines when mounting the volume"

**Comment by bfallonf** [comment]
@ahardin-rh Some comments. I know some of them were there already, but nothing too large.

LGTM otherwise :tada:

File: install_config/storage_examples/gluster_backed_registry.adoc (hunk 4 above)
Comment (ahardin-rh) [inline install_config/storage_examples/gluster_backed_registry.adoc]: Yup! You're right. Good eye.

File: install_config/storage_examples/gluster_backed_registry.adoc (hunk 1 above)
Comment (ahardin-rh) [inline install_config/storage_examples/gluster_backed_registry.adoc]: Thought I got all of these. Thanks for catching these strays!

**Comment by ahardin-rh** [comment]
Rev history in original PR https://github.com/openshift/openshift-docs/pull/1898

**Pull request merged**